# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1314647533 -43200
# Node ID 846f00ceda1ad84481fae9813219a48e06ae54eb
# Parent  a48af224839e4eb2555175d15d703ecd81d0b4c1
trac 9958: patch numerical noise in sage/functions/transcendental.py

diff --git a/sage/functions/transcendental.py b/sage/functions/transcendental.py
--- a/sage/functions/transcendental.py
+++ b/sage/functions/transcendental.py
@@ -79,10 +79,9 @@
     EXAMPLES::
     
         sage: exponential_integral_1(2)
-        0.048900510708061118
+        0.04890051070806112
         sage: w = exponential_integral_1(2,4); w
-        [0.048900510708061118, 0.0037793524098489067, 0.00036008245216265873, 3.7665622843924751e-05] # 32-bit
-        [0.048900510708061118, 0.0037793524098489063, 0.00036008245216265873, 3.7665622843924534e-05] # 64-bit
+        [0.04890051070806112, 0.003779352409848906..., 0.00036008245216265873, 3.7665622843924...e-05]
     
     IMPLEMENTATION: We use the PARI C-library functions eint1 and
     veceint1.
